### PR TITLE
VAGOV-1824 refactor/zero downtime deployment

### PR DESCRIPTION
**Pull request by mbenziane**

### CODE REVIEW

**Comment by mbenziane** [comment]
rebuild

**Comment by ElijahLynn** [comment]
http://internal-devops-vagovcms-3040-201242845.us-gov-west-1.elb.amazonaws.com

File: docker/config/engine/scripts/lando-entrypoint-app.sh [hunk 1]
```diff
@@ -25,14 +25,19 @@ j2 /templates/settings.lando.php.tpl > /app/docroot/sites/default/settings/setti
...
 /usr/bin/wait-for-it.sh -t 120 ${DRUPAL_DATABASE_HOST}:${DRUPAL_DATABASE_HOST_PORT}
+
+if [ "${DRUPAL_DATABASE_HOST}" = "database" ] ; then
 cd ${LANDO_WEBROOT}/vendor/bin/
 ./drush cache:rebuild
 ./drush updatedb -y
 ./drush config:import -y
 ./drush cache:rebuild
+else
+  echo "Skipping vagovcms drupal migrations since we are running it on deploy container" ;
```
Comment (ElijahLynn) [inline docker/config/engine/scripts/lando-entrypoint-app.sh]: Let's change "drupal migrations" to "Drupal update commands".

**Comment by ElijahLynn** [comment]
Please cross link related DevOps PRs in future. 

https://github.com/department-of-veterans-affairs/va.gov-cms-devops/pull/46

**Comment by mbenziane** [comment]
rebuild

**Comment by mbenziane** [comment]
rebuild

**Comment by mbenziane** [comment]
rebuild

**Pull request merged**